components/Spec.vue

```diff
@@ -1,4 +1,7 @@
 <template>
```

Comment:
Is the `template` supposed to be empty here? If so, can we remove the commented code or will we need it?

Reply:
I wasn't sure if we needed the commented code or not - depends on if we are wanting to actually show the text specification to users at some point

Reply:
Can probably safely drop it I think.